### PR TITLE
disoriented title rectified

### PR DESCRIPTION
**Pull request by ankita132**
**Description**
 The title Main Menu appeared disoriented in the introduction page.

Fixes # [ISSUE]
#78 Disoriented Title

**Type of Change**
- Bootstrap tag changed to bring the title in position

**Mocks**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "core-js": "^2.4.1",
     "crypto": "0.0.3",
     "debug": "~2.6.3",
+    "email-verify": "^0.2.1",
```
Comment (paavininanda) [inline package.json]: @ankita132 We don't need to include these changes in this PR.

File: src/app/menu/menu.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="col-md-12 text-center"><h2 class="stage-header">{{language?.menu.heading}}</h2></div>
+<div class="col-md-5 pull-right"><h2 class="stage-header">{{language?.menu.heading}}</h2></div>
```
Comment (paavininanda) [inline src/app/menu/menu.component.html]: Thanks for working on this. Looks good. But it would be good to keep just this change in this PR because that is what corrects the corresponding issue.

Comment (ankita132) [reply src/app/menu/menu.component.html]: But without the other changes, the app cannot be built properly.

Comment (paavininanda) [reply src/app/menu/menu.component.html]: Yes, but once the other PR which is correcting the build issue gets merged, this won't be needed anymore. I'll get back to you after the previous PR gets merged correctly. :)

File: package.json (hunk 1 above)
Comment (shreyans29) [inline package.json]: Why is this added in a PR for CSS changes ?

File: src/app/app.module.ts [hunk 3]
```diff
@@ -92,7 +91,6 @@ import { DoctorService } from './services/doctorchat.service';
         BrowserModule,
         BrowserAnimationsModule,
         DndModule.forRoot(),
-        MaterialModule,
```
Comment (shreyans29) [inline src/app/app.module.ts]: Why is Material Module removed, it works when you install beta 2.0.8 version of Material Module and have the right angular cli version and node version.

File: package.json (hunk 1 above)
Comment (ankita132) [reply package.json]: without installing this package the app won't build.I guess it is required to have the previous PR (#92) to be merged .

File: src/app/app.module.ts (hunk 3 above)
Comment (ankita132) [reply src/app/app.module.ts]: I thought that since material module has deprecated so we should avoid using the module.

**Comment by ankita132** [comment]
Made the necessary changes. The PR can be reviewed after merging PR #92

**Comment by Raxerz** [comment]
@ankita132 It is required that you squash your commits for your PRs.

**Comment by ankita132** [comment]
@Raxerz squashed all the commits into one.

**Comment by shreyans29** [comment]
Hi @ankita132 Take the code from develop branch and check if this issues still exists. Using the proper versions as mentioned in package.json.  If it still exists take an update from the develop branch into your branch for travis to pass.

**Comment by paavininanda** [comment]
@shreyans29 In my local clone it still exists.

**Comment by ankita132** [comment]
@paavininanda I think the problem was with the material module. Using the right versions of Material Module does not give this disoriented title.

**Comment by Buddhiprabha** [comment]
@shreyans29 @paavininanda does the issue addressed by this PR still exist?

**Comment by Raxerz** [comment]
@shreyans29 @paavininanda I am not seeing this issue, can you please verify and close if required.

**Comment by Buddhiprabha** [comment]
@shreyans29 @paavininanda any update?

**Comment by paavininanda** [comment]
I am so sorry for the late reply, but yes this doesn't exist anymore and hence can be closed.
